examples/metatables.clue

```diff
@@ -0,0 +1,4 @@
+local t = {
+    a = 3,
+    meta index = {1+1},
```

Comment:
We should probably use a bigger table with more metamethods

Reply:
I suggest making some sort of basic vector table implementation to show the + metamethods and more

Reply:
hm, I could try

Reply:
since it'll take a bit I'll merge this and update it with a better example later myself